src/org/usfirst/frc4904/robot/commands/ShooterStart.java

```diff
@@ -0,0 +1,19 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class ShooterStart extends CommandGroup {
+	public ShooterStart() {
+		addParallel(new FlywheelSpinup());
```

Comment:
We should probably add the anti-jam somewhere here. Perhaps `addParallel` a `TimedCommand`?